java/script.js

```diff
@@ -45,4 +45,93 @@ const newarrow = document.querySelector('#closebtn');
 
 newarrow.addEventListener('click', () => {
   modalMenu.style.display = 'none';
-});
\ No newline at end of file
+});
+
+const arr1 = [
+  {
+    id: 0,
+    popuptitle: 'Multi-Post Stories 1',
+    popupdescription: 'This is the first A daily for this is a test and is test number 1',
+    popuptec: ['CSS', 'Html', 'Bootstrap', 'Ruby'],
+    popupimg: './pictures/golf.jpg',
+    btnlive: 'https://www.github.com',
+    btnsource: 'https://www.github.com',
+  },
+  {
+    id: 1,
+    popuptitle: 'Multi-Post Stories 2',
+    popupdescription: 'This is the second A daily for this is a test and is test number 2',
+    popupimg: './pictures/beach.jpg',
+  },
+  {
+    id: 2,
+    popuptitle: 'Multi-Post Stories 3',
+    popupdescription: 'This is the third A daily for this is a test and is test number 3',
+    popupimg: './pictures/desert.jpg',
+  },
+];
+
+const arr2 = [
+  {
+    id: 0,
+    popuptec1: 'CSS',
+    popuptec2: 'Html',
+    popuptec3: 'Bootstrap',
+    popuptec4: 'Ruby',
+  },
+];
```

Comment:
[OPTIONAL]

- Good job implementing the suggested changes by the other reviewer 👏 . I think having a separate array for the list of technologies is not necessary because you can add them into the object itself. eg 👇 

`const arr1 = [

  {
    id: 0,
    popuptitle: 'Multi-Post Stories 1',
    popupdescription: 'This is the first A daily for this is a test and is test number 1',
    popuptec: ['CSS', 'Html', 'Bootstrap', 'Ruby'],
    popupimg: './pictures/golf.jpg',
    btnlive: 'https://www.github.com',
    btnsource: 'https://www.github.com',
  },

  {
    id: 1,
    popuptitle: 'Multi-Post Stories 2',
    popupdescription: 'This is the second A daily for this is a test and is test number 2',
    popupimg: './pictures/beach.jpg',
    technologies: [CSS, HTML, Bootstrap, Ruby]
  },

  {
    id: 2,
    popuptitle: 'Multi-Post Stories 3',
    popupdescription: 'This is the third A daily for this is a test and is test number 3',
    popupimg: './pictures/desert.jpg',
    technologies: [CSS, HTML, Bootstrap, Ruby]
  },
];`